pom.xml

```diff
@@ -318,7 +318,7 @@
             <execution>
               <id>ParallelStatsDisabledTest</id>
               <configuration>
-                <reuseForks>true</reuseForks>
+                <reuseForks>false</reuseForks>
```

Comment:
What is this change for?

Reply:
oops, didn't mean to push that one. Makes ParallelStatsDisabledTests more stable when running on a RAM-starved laptop, at the cost of some speed.